charts/pelorus/subcharts/exporters/templates/_imagestream_from_image.yaml

```diff
@@ -20,13 +20,13 @@ spec:
       name: {{ .image_name }}
     # .image_name is provided without tag
     {{- else }}
-      name: {{ .image_name }}:{{ .image_tag | default "2.0.7" }}
+      name: {{ .image_name }}:{{ .image_tag | default "v2.0.7" }}
     {{- end }}
 {{- else }}
-      name: quay.io/pelorus/pelorus-{{ .exporter_type }}-exporter:{{ .image_tag | default "2.0.7" }}
+      name: quay.io/pelorus/pelorus-{{ .exporter_type }}-exporter:{{ .image_tag | default "v2.0.7" }}
```

Comment:
this one I agree on adding `v` (as it is how it is on quay). But is it necessary for the other places?

Reply:
yes and now, will explain in detail offline